Sources/ProjectDescription/Dependencies/DependenciesGraph.swift

```diff
@@ -0,0 +1,17 @@
+/// A directed acyclic graph (DAG) that Tuist uses to represent the dependency tree.
+public struct DependenciesGraph: Equatable, Codable {
```

Comment:
I understand why this is part of `ProjectDescription` - however, bear in mind that everything public in `ProjectDescription` module can be used by users. Therefore, I think it should be moved elsewhere. Either `TuistCore` or `TuistGraph` probably

Reply:
I have moved it to the `TuistDependencies` module as it is used only in the `tuist dependencies` command.
It only "leaks" to the `TuistLoader` module because the `ManifestModelConverting` defines the method to convert from `TuistDependencies.DependenciesGraph` to `TuistGraph.DependenciesGraph`. A possible workaround would be to move the converter inside `TuistDependencies`, but then it would require `TuistDependencies` to depend on `TuistLoader` to convert the `ProjectDescription` manifests contained in `TuistDependencies.DependenciesGraph` to the ones contained in `TuistGraph.DependenciesGraph`.

Let me know if you are fine with the current solution, or if instead you would prefer to move it somewhere else

Reply:
what do you think about moving it `TuistCore` and adding `ProjectDescription` dependency to that module? This might be the "nicest" solution - but I'd probably be fine with the current one as well if that is not possible

Reply:
Moved in this commit: https://github.com/tuist/tuist/pull/3099/commits/00bdf78659f7c22d67b80e1104754f01087accd7